src/util/customTransformer-v1.js

```diff
@@ -62,6 +63,9 @@ async function userTransformHandlerV1(
     userTransformation.code,
     libraryVersionIds,
     userTransformation.versionId,
+    userTransformation.id,
+    userTransformation.workspaceId || events[0].metadata?.workspaceId,
```

Comment:
```suggestion
    userTransformation.workspaceId,
```
this is enough

Reply:
`userTransformation.versionId` can be potentially be removed after this change. Lets do it in next PR